XenModel/Messages.resx

```diff
@@ -6235,6 +6235,11 @@ Note: Any custom color selections will remain unchanged</value>
   <data name="GUEFI_SECURE_BOOT_MODE_LABEL" xml:space="preserve">
     <value>UEFI &amp;Secure Boot</value>
   </data>
+  <data name="GUI_NOT_COMPATIBLE" xml:space="preserve">
+    <value>This version of [XenCenter] can only connect to {0} 7.1 CU2 and [XenServer] 8.2 and greater versions and cannot connect to {1}.

```

Comment:
The first sentence has too many ands. Can we perhaps say `This version of [XenCenter] can only connect to {0} 7.1 CU2, [XenServer] 8.2, and greater versions. It cannot connect to {1}.`

Reply:
Fixed.